schemas/v1.0/schema.json

```diff
@@ -1,7 +1,7 @@
 {
     "$id": "https://spec.openapis.org/arazzo/1.0/schema/2024-08-01",
```

Comment:
we may also want to consider the `WORK-IN-PROGRESS` for schema development purposes.  Overlay and OAS have included changing this to the deployment date with a build script on merge